tests/longevity-tests/Dockerfile.6.5

```diff
@@ -15,4 +15,4 @@
 FROM longevity-base
 ENV GOVC_URL=blinky.eng.vmware.com
 ENV TEST_URL_ARRAY=blinky.eng.vmware.com
-ENV STATIC_VCH_OPTIONS="--insecure-registry=vic-executor1.vcna.io --public-network-ip 10.17.109.7/24 --public-network-gateway 10.17.109.253 --dns-server 10.118.81.1"
+ENV STATIC_VCH_OPTIONS="--insecure-registry=vic-executor1.vcna.io --public-network-ip 10.197.37.210/23 --public-network-gateway 10.197.37.253 --dns-server 10.142.7.21"
```

Comment:
I thought we wanted the executors to be on onecloud? Does this move them off?

Reply:
no, that is the static IP address that the VCH gets assigned nothing to do with executors.